docs/mergeduty/howto.md

```diff
@@ -198,6 +197,7 @@ Diff should be similar to [this one](https://hg.mozilla.org/releases/mozilla-esr
 Make sure the bug that tracks the migration has no blocking items.
 
 ### Land whatsnewpage list of locales
+**TODO** - this needs to change, as the process no longer assumes this, but apply them; the l10n drivers provide the final list of locales to receive the WNP on the Tuesday prior to the ship date.
```

Comment:
we always use the friday before first merge day as the due date for locales. which tues is this for now?

Reply:
Let's talk more at the post-mortem, I add more issues there. But Tuesday as in tomorrow.